analyzer/consensus/consensus.go

```diff
@@ -36,6 +38,12 @@ import (
 
 const (
 	consensusAnalyzerName = "consensus"
+
+	// The query takes ~20 second to run on mainnet at the time of implementing this.
+	accountListViewRefreshInterval = 2 * time.Minute
```

Comment:
Callout to explorer folks that the `consensus/accounts` endpoint might return slightly stale (<2min) info, but we don't expect this to be an issue since the `consensus/account/{address}` endpoint used for account details view always returns the latest info. @lukaw3d @csillag @lubej